JSON_for_IO-Link.yaml

```diff
@@ -1030,17 +1048,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/masterCapabilitiesGet'
-              examples:
-                wired:
-                  value:
-                    numberOfPorts: 8
```

Comment:
I disagree on this deletion. Why should the GET/masters command not return the fact wether it is a wired or a wireless master ?

Reply:
@mrentsch65 Wired and wireless Master has the same return values. So I thought it didn't matter. What do you prefer to return?

portType*	string
Enum:
[ CLASS_A, CLASS_B, CLASS_A_WITH_PORT_POWER_OFF_ON, FAILSAFE_PORT_A_WITHOUT_SAFETY_DIGITAL_INPUTS, FAILSAFE_PORT_A_WITH_SAFETY_DIGITAL_INPUTS, FAILSAFE_PORT_B, WIRELESS_MASTER ]

Reply:
Would the wireless master not just return numberOfPorts, but not:
  "maxPowerSupply": {
    "value": 0.3,
    "unit": "A"
  }
?

Reply:
That is the maximum power consumption of the entire master or am I understood it wrong? So that can perhaps also be interesting for wireless, even if the value is of course much lower. Only as information that the customer knows what kind of power supply he should use.

Reply:
@fm-sick : Is this not the power supply capability of a single port?

Reply:
How I see is the power supply of single port placed in:
/masters/{masterNumber}/ports/{portNumber}/capabilities